package.json

```diff
@@ -13,7 +13,7 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && mocha --timeout 20000"
+    "test": "if-node-version \">=4\" xo && mocha --timeout 20000"
```

Comment:
If it was me I'd used raw eslint and be done with backward compat issue. But that's fine.